sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 #### Bugs Fixed
 
 #### Other Changes
+- Made `auth.py` a private file, making the GetAuthorizationMethod no longer public
```

Comment:
This immediately strikes me as a breaking change--removing public surface area.

Reply:
I was torn between that and this since we didn't export the auth file or methods anywhere - but if it's a breaking change regardless can I just remove the authorization method rather than deprecating it?

Reply:
We can't break it... we can really only throw deprecation warnings around...
There might be ways to mess with dunder get_attr in the init file to support backwards compatibility... but we haven't experimented with this to know what it's limitations/drawbacks would be.

Reply:
ill make the file public again and just stick to the current deprecation message in that case